ncr/src/main/kotlin/com/hartwig/actin/personalization/ncr/interpretation/filter/RecordFilter.kt

```diff
@@ -1,15 +1,19 @@
 package com.hartwig.actin.personalization.ncr.interpretation.filter
 
 import com.hartwig.actin.personalization.ncr.datamodel.NcrRecord
+import io.github.oshai.kotlinlogging.KotlinLogging
 
 interface RecordFilter {
     val logFilteredRecords: Boolean
     
     fun log(message: String) {
         if (logFilteredRecords) {
-            println(message)
+            val logger = KotlinLogging.logger {}
+            logger.info { message }
         }
     }
     
-    fun apply(record: List<NcrRecord>): Boolean
+    fun List<NcrRecord>.tumorId() = first().identification.keyZid
+    
+    fun tumorRecords(record: List<NcrRecord>): Boolean
```

Comment:
Sorry, my comment wasn't clear. I meant to suggest renaming the parameter:
```suggestion
    fun apply(tumorRecords: List<NcrRecord>): Boolean
```